reana_workflow_engine_yadage/externalbackend.py

```diff
@@ -82,6 +82,8 @@ def submit(self, spec, parameters, state, metadata):  # noqa: C901
         resources = spec["environment"].get("resources", None)
         if resources:
             for item in resources:
+                if not isinstance(item, dict):
+                    continue
```

Comment:
This works well, but the user is not informed of the situation... It may be good to inform the user in the logs that REANA necessitates a dictionary and that the current value is not formatted that way and so is being ignored...